### ISSUE TITLE
Rename the tool

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-em-report-tool#232]
Something along the lines of "WCAG-EM report Generator" or "WCAG Evaluation Assistant". The current name is confusing users about its purpose. This issue should be worked on together with the Education & Outreach Working Group.

**Comment by davidberman** [comment]
I agree it would benefit from a better name. "report generator" makes more sense, as it clarifies what it does, while "evaluation assistant" wrongly implies the tool will perform checks.

Whatever name is selected, I think it essential that it be clearer that the tool is <em>not</em> required to produce a WCAG-EM compliant report ... rather it one way you could choose to go about creating such a report.

**Comment by rvantonisse** [comment]
Related #127

**Comment by hidde** [comment]
We'll soon release a new tool called [ATAG Report Tool](https://github.com/w3c/wai-atag-report-tool), I would vote for keeping WCAG-EM Report Tool as a name, to establish a clear relationship between the tools.

**Comment by nitedog** [comment]
EOWG decided to keep the name for several reasons, including consistency with ATAG Report Tool.
